.github/workflows/build.yml

```diff
@@ -47,5 +56,24 @@ jobs:
         run: |
             sudo apt-get update
             sudo apt-get install ghostscript -y
-      - name: Build & Test
+
+      - name: Basic Auth Build & Test
+        env:
+          - FINERACT_SECURITY_BASICAUTH_ENABLED: true
+          - FINERACT_SECURITY_OAUTH_ENABLED: false
```

Comment:
Do these variables actually do anything? It seems that we are hardcoding the same parameters in the command line when starting the server for oauth2-test and basictest etc? So it would seem that these are redundant... correct?

Reply:
Hm, now that you mention it... actually no... I was starting with a different idea and then I guess I forgot to remove them. Good catch!